unit_test/CMakeLists.txt

```diff
@@ -90,16 +90,16 @@ IF (Kokkos_ENABLE_OpenMP)
   
   APPEND_GLOB(OPENMP_BLAS_SOURCES ${CMAKE_CURRENT_SOURCE_DIR}/openmp/Test_OpenMP_Blas*.cpp)
 
-  IF (KOKKOS_ENABLE_DEBUG)
-    SET(DISABLE_SLOW_DGEMM_DOUBLE_TEST "--gtest_filter=-openmp.gemm_double")
-  ENDIF()
+  # IF (KOKKOS_ENABLE_DEBUG)
+  #   SET(DISABLE_SLOW_DGEMM_DOUBLE_TEST "--gtest_filter=-openmp.gemm_double")
```

Comment:
Are we re-enabling this test in debug mode ? Is this planned.

Reply:
We enable this test from Trilinos after we reduce the test problem sizes. This is matching modification that we already made in Trilinos.

Reply:
Got it, Thanks !